Frontend/fcc.config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
 
   // root uri of the API
-  apiRoot: "http://localhost:5000/",
+  // apiRoot: "http://35.197.238.123/api",
+  apiRoot: "http://127.0.0.1:8000/",
```

Comment:
@Aibaki The assignment of `apiRoot: "http://127.0.0.1:8000/",` is only applicable to the docker specific deployment method and won't work for the Ubuntu deployment.

Please identify it in the deployment doc as a mandatory change pre-deployment and consider making it non-specifi, eg. `apiRoot: "http://<api-address>:<api-port>/",`